sherpa/bin/web/js/offline_record.js

```diff
@@ -21,8 +21,10 @@ function initWebSocket() {
 
   // Listen for messages
   socket.addEventListener('message', function(event) {
-    document.getElementById('results').value = event.data;
-    console.log('Received message: ', event.data);
+    var result='';
```

Comment:
Can you remove the changes from this file?

Reply:
ok，removed.

Reply:
Looks like you removed the whole file. Could you undo the changes?

Reply:
Sorry,Undo changes. Look again

Reply:
Could you push you new changes?